qiskit/circuit/library/standard_gates/equivalence_library.py

```diff
@@ -289,6 +291,52 @@
     def_iswap.append(inst, qargs, cargs)
 _sel.add_equivalence(iSwapGate(), def_iswap)
 
+# SXGate
+
+q = QuantumRegister(1, 'q')
+def_sx = QuantumCircuit(q)
+for inst, qargs, cargs in [
+        (RZGate(-pi / 2), [q[0]], []),
```

Comment:
better to use non-parametric gates in non-parametric equivalences, and makes the Clifford structure more apparent as well.
```suggestion
        (SdgGate(), [q[0]], []),
```